rabbit/spring-cloud-starter-stream-sink-rabbit/src/main/java/org/springframework/cloud/stream/app/rabbit/sink/RabbitSinkConfiguration.java

```diff
@@ -86,7 +86,7 @@ public RabbitTemplate rabbitTemplate(ConnectionFactory rabbitConnectionFactory)
 	}
 
 	@Bean
-	@ConditionalOnProperty(name = "converterBeanName", havingValue = RabbitSinkProperties.JSON_CONVERTER)
+	@ConditionalOnProperty(name = "rabbit.converterBeanName", havingValue = RabbitSinkProperties.JSON_CONVERTER)
```

Comment:
I'd really prefer `prefix` attribute of this annotation instead.
Not sure if that will work if we present the property like though:
```
rabbit.converter-bean-name
...
RABBIT_CONVERTER_BEAN_NAME
```
I mean that there might was a reason to introduce that `prefix` anyway in Boot.